p2p/src/protocol.rs

```diff
@@ -353,9 +353,16 @@ impl MessageHandler for Protocol {
 
 				Ok(None)
 			}
-
-			_ => {
-				debug!("unknown message type {:?}", msg.header.msg_type);
+			Type::Error => {
```

Comment:
You can combine checks `Type::Error | Type::Hand | Type::Shake`

Reply:
:+1: I had it broken out explicitly like this when making these changes so it was easier to see all msg types in there. But I'll merge these like you suggest.